modules/user_data/templates/start_graphdb.sh.tpl

```diff
@@ -144,7 +144,15 @@ aws --cli-connect-timeout 300 ssm get-parameter --region ${region} --name "/${na
 
 graphdb_cluster_token="$(aws --cli-connect-timeout 300 ssm get-parameter --region ${region} --name "/${name}/graphdb/cluster_token" --with-decryption | jq -r .Parameter.Value)"
 
-cat << EOF > /etc/graphdb/graphdb.properties
+aws --cli-connect-timeout 300 ssm get-parameter --region ${region} --name "/${name}/graphdb/graphdb_config" --with-decryption | jq -r .Parameter.Value | \
+  base64 -d > /etc/graphdb/graphdb.properties
+
+
+GDB_CONFIG="$(aws --cli-connect-timeout 300 ssm get-parameter --region ${region} --name "/${name}/graphdb/GDB_JAVA_OPTS" --with-decryption | jq -r .Parameter.Value)"
+
+echo "\$GDB_JAVA_OPTS $GDB_CONFIG" >> /etc/environment.d/gdb_environments
```

Comment:
it's probably best to move this file in `/etc/graphdb/graphdb.env` @mihailradkov ?